device/samples/simple_sample_device_x509.js

```diff
@@ -4,17 +4,17 @@
 'use strict';
 
 var fs = require('fs');
-var Protocol = require('azure-iot-device-mqtt').Mqtt;
+//var Protocol = require('azure-iot-device-mqtt').Mqtt;
 // Uncomment one of these transports and then change it in fromConnectionString to test other transports
 // var Protocol = require('azure-iot-device-amqp').AmqpWs;
-// var Protocol = require('azure-iot-device-http').Http;
+var Protocol = require('azure-iot-device-http').Http;
```

Comment:
Why are you setting the default to HTTP instead of MQTT?

Reply:
Because I was being sloppy.  I shall remove that and make it default to MQTT.